app/src/main/java/com/nervousfish/nervousfish/activities/RhythmCreateActivity.java

```diff
@@ -137,14 +137,15 @@ public void onDoneCreatingRhythmClick(final View v) {
             final KeyPair keyPair = profile.getKeyPairs().get(0);
 
             LOGGER.info("Sending my profile with name: {}, public key: {}", profile.getName(), keyPair.getPublicKey().toString());
-            final Contact myProfileAsContact = new Contact(profile.getName(), new Ed25519Key("Ed25519 key", "73890ien"));
+            final Contact myProfileAsContact = new Contact(profile.getName(),
+                    new Ed25519Key("Ed25519 key", "73890ien"), profile.getIban());
```

Comment:
Can't you get the key from the database? If not, can you open an issue for that?